nextflow.config

```diff
@@ -115,7 +115,7 @@ params {
     bgc_skip_deepbgc                    = false
     bgc_deepbgc_database                = null
     bgc_deepbgc_score                   = 0.5
-    bgc_deepbgc_prodigalmetamode        = true
+    bgc_deepbgc_prodigalsinglemode        = true
```

Comment:
```suggestion
    bgc_deepbgc_prodigalsinglemode        = false
```

Reply:
If we have `annotation_prokka_singlemode` set to `false` to activate the meta mode we should also have it here for `prodigal` so that should be `false` to also activate the metagenome mode

Reply:
Agreed!